charts/kaskada-canary/tests/__snapshot__/default_test.yaml.snap

```diff
@@ -0,0 +1,152 @@
+manifest should match snapshot:
```

Comment:
these snapshots are the "proof" that the chart is working as designed.  we test against these in CI.